modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+= Tasks provided in the {pipelines-shortname} namespace
+
+A {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortnames} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
+
+`ClusterTask` functionality is deprecated since version 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
+
+You cannot specify a custom execution image in any of the tasks available in the {pipelines-shortname} installation namespace. These tasks do not support parameters such as `BUILDER_IMAGE`, `gitInitImage`, or `KN_IMAGE`. If you want to use a custom execution image, create a copy of the task and replace the image by editing the copy.
+
+[discrete]
+[id="op-taskref-buildah_{context}"]
+== buildah
+
+The `buildah` task builds a source code tree into a container image and then pushes the image to a container registry.
+
+.Example usage of the `buildah` task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: build-and-deploy
+spec:
+# ...
+  tasks:
+# ...
+  - name: build-image
+    taskRef:
+      resolver: cluster
+      params:
+      - name: kind
+        value: task
+      - name: name
+        value: buildah
+      - name: namespace
+        value: openshift-pipelines
+    params:
+    - name: IMAGE
+      value: $(params.IMAGE)
+    workspaces:
+    - name: source
+      workspace: shared-workspace
+# ...
+----
+
+.Supported parameters for the `buildah` task
+[options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`IMAGE` |Fully qualified container image name to be built by Buildah. |`string` |
+|`DOCKERFILE` |Path to the `Dockerfile` (or `Containerfile`) relative to the `source` workspace. |`string` |`./Dockerfile`
+|`CONTEXT` |Path to the directory to use as the context. |`string` |`.`
+|`STORAGE_DRIVER` |Set the Buildah storage driver to reflect the settings of the current cluster node settings. |`string` |`vfs`
+|`FORMAT` |The format of the container to build, either `oci` or `docker`. |`string` |`oci`
+|`BUILD_EXTRA_ARGS` |Extra parameters for the `build` command when building the image. |`string` |
+|`PUSH_EXTRA_ARGS` |Extra parameters for the `push` command when pushing the image. |`string` |
+|`SKIP_PUSH` |Skip pushing the image to the container registry. |`string` |`false`
+|`TLS_VERIFY` |The TLS verification flag, normally `true`. |`string` |`true`
+|`VERBOSE` |Turn on verbose logging, all commands executed are added to the log. |`string` |`false`
+|===
+
+.Supported workspaces for the `buildah` task
+[options="header"]
```

Comment:
Same here about giving more width to the description column. I won't point out any more tables if you could look through the rest in this file as well.